kotsadm/pkg/online/online.go

```diff
@@ -229,6 +229,17 @@ func CreateAppFromOnline(pendingApp *types.PendingApp, upstreamURI string, isAut
 		if err := preflight.Run(pendingApp.ID, pendingApp.Slug, newSequence, false, tmpRoot); err != nil {
 			return nil, errors.Wrap(err, "failed to start preflights")
 		}
+		isUpdate := false
+		if err := reporting.SendPreflightInfo(pendingApp.ID, int(newSequence), skipPreflights, isUpdate); err != nil {
```

Comment:
this seems to be duplicated by the one on line 239, since the function is called regardless if preflights are skipped or not